### DISCUSSION TITLE
Release policy

### DISCUSSION THREAD

**Discussion by amitkot** [LunarVim/LunarVim#4518] [General]
Hi 👋 
I've been using LunarVim for over a year now and enjoying it greatly!
I switched from configuring NeoVim on my own and I find the choices and decisions implemented in LunarVim are a great fit for my needs.

I'm running LunarVim 1.3 and there have been many changes in the past couple of months (besides NeoVim minor versions, null-ls replaced by none-ls, etc.).

What is the release policy for LunarVim? If I want to use "relatively" up-to-date versions, do I have an option besides switching to Nightly?

Thanks 🙏 
Amit

**Comment by LostInTheLogs** [comment]
Hi, sorry for the late reply,
Since I moved to astronvim, there's no one actively working on lunarvim, so there probably won't be another release for quite a while (the pinned core plugins aren't being updated even on nighlty) I'd switch to nightly if I were you

**Reply by amitkot** [reply]
@LostNeophyte thanks for your reply! I'll take a look at Astronvim 🙏

**Reply by dimitarvp** [reply]
@LostNeophyte IMO you should be more open about it and put a warning at the top of the README. Something like "LunarVim is looking for maintainers, currently development pace is greatly slowed down" or something along the lines.

I was bitten by https://github.com/LunarVim/LunarVim/issues/3680 and after trying everything inside it absolutely nothing worked. If I will be using `LVIM_DEV_MODE=1` then I might as well move to AstroNvim as well, no point in having Lazy update almost everything and then LunarVim spits out 100+ errors on startup. 😕 

Anyhow, thanks for at least posting the news here.

**Reply by LostInTheLogs** [reply]
> @LostNeophyte IMO you should be more open about it and put a warning at the top of the README. Something like "LunarVim is looking for maintainers, currently development pace is greatly slowed down" or something along the lines.

I enjoyed being a maintainer of the project, but unfortunately i don't have that much free time anymore so I slowly stopped contributing, I thought that someone else wold take over but that didn't happen. I'll write a message on our dev discord channel and talk about it with the other devs
 
> I was bitten by #3680 and after trying everything inside it absolutely nothing worked. If I will be using `LVIM_DEV_MODE=1` then I might as well move to AstroNvim as well, no point in having Lazy update almost everything and then LunarVim spits out 100+ errors on startup. 😕

That issue is caused by the treesitter queries bundled with your version of neovim you can delete them or use an older version of neovim. IIRC we did set the path correctly so the correct ones should be used but for some reason sometimes it doesn't work. It worked on my machine I was out of ideas.

**Reply by WakeUpHugo** [reply]
Went to check astronvim. Any reason you switched over to that other distro? @LostNeophyte

**Reply by LostInTheLogs** [reply]
> Went to check astronvim. Any reason you switched over to that other distro? @LostNeophyte

The maintainer is very active and I like the way its configured (in v3) - it's pretty much just lazy.nvim plugin specs. It also has a lot of preconfigured plugins in astrocommunity that can be easily imported to the config

**Reply by dimitarvp** [reply]
For my part I use AstroNvim for a week now and the thing I dig the most is how actively it's maintained + how much choice you have with AstroCommunity. Everything in there is opt-in, you don't have to get all plugins if you don't need them. You can just check them out and enable them one by one and try them out like I did. This really helped me iterate on my config and make it truly my own.

**Reply by thezacharytaylor** [reply]
Jumping on the bandwagon after reading this thread... it's fantastic how Astro offers many familiar features and keybinds. While there are some differences, they're incredibly user-friendly, so you might find yourself keeping them. The file structure reminds me a lot of a vanilla NeoVim setup, which is great, and the documentation is truly exceptional. It even allowed me to streamline my setup honestly! I'm still tinkering with it a bit to personalize things and make sure I'm only using the features I need. Overall, I'm super impressed and would definitely recommend it!

**Reply by nithin-mk** [reply]
So, which distro is best for a newbie like me - AstroNvim, LazyVim, or NvChad? :-)

**Reply by Zeioth** [reply]
LostNeophyte Check out [NormalNvim](https://github.com/NormalNvim/NormalNvim). Originally I forked it from AstroNvim to be more like what LunarVim was trying to achieve. But now it has many unique features.

**Reply by hamzamohdzubair** [reply]
Being a user of Lunarvim for 2+ years now and the only distro i have ever used and been comfortable with, i find it sad that you have decided to move on from lunarvim. And I also completely respect your decision to do so. However, I also hope that the other 303 contributors will keep making valuable contributions to this project.

**Reply by MDSADABWASIM** [reply]
Better than switching, we should maintain lunarvim.

**Reply by cassepipe** [reply]
Been using LazyVim for two months because I could not make AstroNvim work with nvim APPNAME for some reason
It's okayish but  I must say I *miss* Lunarvim a lot. It was just as powerful but simpler.

**Reply by reisnobre** [reply]
I've tried LazyVim for a couple weeks. Vue setup was similar but worked worse. 
I'm thinking about starting doing some work LunarVim because I really like how stable it is and how good it is with my current workflow.
After using LazyVim I've found some nice points and things to learn from there. 
I don't know If I'll be successful on updating some plugins and adding the next generation of plugins like the ones found on https://github.com/folke/snacks.nvim and https://github.com/echasnovski/mini.nvim. But on the next days I'll organize a list of wants and needs.
